src/script/bitcoinconsensus.cpp

```diff
@@ -69,7 +69,7 @@ struct ECCryptoClosure
 ECCryptoClosure instance_of_eccryptoclosure;
 }
 
-int bitcoinconsensus_verify_script(const unsigned char *scriptPubKey, unsigned int scriptPubKeyLen, uint64_t amount,
+int bitcoinconsensus__verify_script(const unsigned char *scriptPubKey, unsigned int scriptPubKeyLen, CAmount amount,
```

Comment:
CAmount is an internal type, so certainly don't expose it. If you don't expose it, it can be static :)